src/main/java/com/java/dojo/recipes/entities/Comment.java

```diff
@@ -0,0 +1,26 @@
+package com.java.dojo.recipes.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
```

Comment:
Isso serve para todas as classes:

Eu acho que a gente não deveria usar @Data nas entidades do JPA por que tem algumas chances de ter conflitos no futuro, essa annotation gera @ToString, @EqualsAndHashCode, @RequiredArgsConstructor e também @Getter e @Setter. 

Tem um artigo legal falando sobre os "riscos" de algumas anotações do JPA com Lombok:

https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/

Em relação ao @AllArgsConstructor eu acho que a gente poderia colocar acesso private nele, visto que a gente vai usar o @Builder então não vamos utilizar em outras partes da aplicação um construtor com os parametros da classes, mas o builder pattern:

`@AllArgsConstructor(access = AccessLevel.PRIVATE)`

Eu acho que a gente pode colocar o id dessas classes pra serem auto generated, assim cada objeto salvo gera um id automaticamente... uma forma seria:

`	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;`

Reply:
Bem demais Pedrão, eu também não manjo muito de utilizar o lombok mas vou correr atrás pra ficar por dentro!